up-l3/usubscription/v3/README.adoc

```diff
@@ -18,139 +18,615 @@ SPDX-FileType: DOCUMENTATION
+--
 
+[.specitem,oft-sid="req~usubscription-state-remote-sub-response-positive~1",oft-needs="impl,utest"]
+--
+When receiving a positive response to the forwarded subscription request, the local uSubscription service *MUST* set the topic state to `SUBSCRIBED`.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-sub-response-negative~1",oft-needs="impl,utest"]
+--
+When receiving a negative response to the forwarded subscription request, the local uSubscription service *MUST* set the topic state to `UNSUBSCRIBED`.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub~1",oft-needs="impl,utest"]
+--
+When the last Subscriber uE has called Unsubscribe(topic) to a remote topic, local uSubscription service *MUST* forward an unsubscribe request to the destination device uSubscription service.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub-state~1",oft-needs="impl,utest"]
+--
+When local uSubscription service forwards an unsubscribe request to the destination device uSubscription service, it *MUST* set the topic state to `UNSUSBCRIBE_PENDING`.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub-response-positive~1",oft-needs="impl,utest"]
+--
+When receiving a positive response to the forwarded unsubscribe request, the local uSubscription service *MUST* set the topic state to `UNSUBSCRIBED`.
+--
+
+[.specitem,oft-sid="req~usubscription-state-remote-unsub-response-negative~1",oft-needs="impl,utest"]
+--
+When receiving a negative response to the forwarded unsubscribe request, the local uSubscription service *MUST* set the topic state to `UNSUBSCRIBED_PENDING`.
+--
 
 == Subscription Change Notifications
 
-The uSubscription service notifies observers when there is a change in a subscription states by publishing notification events to the topic `/core.usubscription/3/subscriptions#Update`.  Below are the specific nuances about the subscription change notification based on the observer type.
+=== Default notification topic
+
+As has been described in the xref:usubscription-local-remote-subscriptions[remote subscriptions section] above, uProtocol dispatchers (`uStreamers`) are expected to intelligently forward messages to remote systems in cases where a remote uEntity (and thus a remote uSubscription service instance) made a subscription request to our local uProtocol network. 
+
+[.specitem,oft-sid="dsn~usubscription-change-notification~1",oft-needs="impl,utest"]
+--
+To support this use case, uSubscription service *MUST* make all subscription changes available as event messages, published on a topic specifically defined for this purpose.
+--
 
-===  Subscribers
+[.specitem,oft-sid="dsn~usubscription-change-notification-type~1",oft-needs="impl,utest"]
+--
+Subscription change messages *MUST* be of type https://github.com/eclipse-uprotocol/up-spec/blob/da5ca97d3a7541d2fcd52ed010bc3bcca92e46cb/up-core-api/uprotocol/core/usubscription/v3/usubscription.proto#L302[Update].
```

Comment:
Please see [usubscription.proto](https://github.com/eclipse-uprotocol/up-spec/blob/6b8c38b378daf9f4b8623b45e692e67517007a40/up-core-api/uprotocol/core/usubscription/v3/usubscription.proto#L38), the name is not the same but line 38 defines the explicit notification topic for subscription changes

Reply:
That's true, but this is not about the topic but the Update message type.

Reply:
see previous clarification